chrome_manifest_v3/service_worker.js

```
@@ -1,79 +1,79 @@
-var CLEAR_EXT_BROWSING_DATA = "g2t_clear_extension_browsing_data";
-var UPLOAD_ATTACH = "g2t_upload_attach";
-var UPLOAD_ATTACH_STORE = "g2t_upload_attach_store";
-var UPLOAD_ATTACH_RESULTS = "g2t_upload_attach_results";
+var CLEAR_EXT_BROWSING_DATA = 'g2t_clear_extension_browsing_data';
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)